pkgs/applications/networking/cluster/habitat/default.nix

```
@@ -1,24 +1,22 @@
 { lib, fetchFromGitHub, rustPlatform, pkgconfig
```

Add dependencies for darwin

```suggestion
 { stdenv, lib, fetchFromGitHub, rustPlatform, pkgconfig, darwin
```